src/schema/utils.ts

```diff
@@ -185,3 +186,22 @@ export const isEmptyObject = (o: any) => isDeepStrictEqual(o, {})
 export function keys<A extends object>(a: A): (keyof A)[] {
   return Object.keys(a) as any
 }
+
+export function ensureDepIsInstalled(depName: string) {
+  try {
+    require(depName)
+  } catch (err) {
+    if (err.code === 'MODULE_NOT_FOUND') {
+      console.error(
+        `${chalk.redBright('ERROR:')} ${chalk.greenBright(
+          depName
+        )} must be installed for nexus-plugin-prisma to work properly. Please run \`${chalk.greenBright(
+          `npm install ${depName}`
```

Comment:
Suggestion:

> When nexus-plugin-prisma is used as a Nexus schema plugin then you must have ... installed.

Point: Can we lower confusion by qualifying this is just for schema plugin use-case.

Reply:
![image](https://user-images.githubusercontent.com/25233379/86145567-5bea6d80-baf7-11ea-96d4-42cce4eb43ef.png) All good ?